src/styles/common.css

```diff
@@ -93,6 +94,7 @@ textarea {
   border-radius: 0.3em;
   border: 2px solid #ebf1fa;
   background-color: #ffffff;
+  min-width: 250px;
```

Comment:
Just because the text area element wasn't taking any of the empty surrounding space and looked weirdly small in the layout.

Before:

<img width="1428" alt="before" src="https://user-images.githubusercontent.com/88476223/129209388-5b08e271-1d59-4c7a-9b39-dc2cd17c5bac.png">

After:

<img width="1439" alt="after" src="https://user-images.githubusercontent.com/88476223/129209465-d86027f6-443a-4e71-8d5d-a363013a610a.png">